src/components/Users/Users.jsx

```diff
@@ -1,9 +1,7 @@
 import React from "react";
-import userPhoto from "../../assets/images/user.png";
-import { NavLink } from "react-router-dom";
 import Pagination from "@material-ui/lab/Pagination";
-import Button from "@material-ui/core/Button";
 import styled from "styled-components";
+import User from "./User";
 
 const Users = ({
```

Comment:
Just try to understand why you need the logic  
`let pages = [];`
 `for (let i = 1; i <= pagesCount; i++) {`
    `pages.push(i);`
 ` }`